### DISCUSSION TITLE
loading false during fetchmore

### DISCUSSION THREAD

**Discussion by sve-odoo** [Akryum/vue-apollo#466] [Q&A]
Hello,

When fetchMore is running, $apollo.queries.myquery.loading is false.

Other people have had this problem with apollo-client and solved it with 
`notifyOnNetworkStatusChange: true`
( see https://github.com/apollographql/apollo-client/issues/1617 or https://github.com/apollographql/react-apollo/issues/727 )

But it doesn't seem to solve the problem in vue-apollo

Thanks in advance for your help.

**Update:**
loading does become true; but only in the component that called the fetchMore
In other components using this query, loading is true during the first fecthing, but not during the fetchMore.

**Comment by jedikcz** [comment]
This is also happening, if a component loads data via 

`apollo: {
   query: ...,
   variables() {
    return {id: someReactiveProperty}
   }}
`

and _someReactiveProperty_ gets changed within the component. Query is correctly rerun, but loading state remains false.

**Comment by neilpoulin** [comment]
I had a similar issue with react-apollo, and it was "fixed" by setting notifyOnNetworkStatusChange=true on the Query component. 

I swear this has worked on other queries in my project before without adding that flag, but it's working now so i'm happy.
